src/SwedbankPay/Core/Adapter/WC_Adapter.php

```diff
@@ -258,7 +258,7 @@ public function getOrderData($orderId)
             // Get Product Class
             $productClass = get_post_meta($orderItem->get_product()->get_id(), '_sb_product_class', true);
             if (empty($productClass)) {
-                $productClass = 'ProductGroup1';
+                $productClass = apply_filters('sb_product_class', 'ProductGroup1', $orderItem->get_product());
```

Comment:
Is `sb_` already esablished as a prefix for our filters? If not, I'd like them changed to `sbp_`.

```suggestion
                $productClass = apply_filters('sbp_product_class', 'ProductGroup1', $orderItem->get_product());
```

Reply:
@asbjornu Sorry, what do you mean about "not our filters"? There are filters which allows WordPress developers to override product classes

Reply:
Please read what I wrote again. There is no "not" in my sentence. I understand why the filters are there, what I'm asking is about the `sb_` prefix in the filter's name.

Reply:
@asbjornu I think we should leave `sb_` prefix